x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackSettings.java

```diff
@@ -134,6 +134,10 @@ private XPackSettings() {
     /** Setting for enabling or disabling vectors. Defaults to true. */
     public static final Setting<Boolean> VECTORS_ENABLED = Setting.boolSetting("xpack.vectors.enabled", true, Setting.Property.NodeScope);
 
+    /** Setting for enabling or disabling aggregate_metric fields. Defaults to true. */
+    public static final Setting<Boolean> AGGREGATE_METRIC_ENABLED = Setting.boolSetting("xpack.aggregate_metric.enabled",
```

Comment:
Since this is targeting Basic, I don't believe we need the `enabled` flag and associated code (e.g. that's what we did with the `analytics` plugin: https://github.com/elastic/elasticsearch/pull/46092/)

Reply:
Fixed